tools/bug-report/pkg/content/content.go

```diff
@@ -154,18 +140,43 @@ func GetIstiodInfo(p *Params) (map[string]string, error) {
 		return nil, fmt.Errorf("getIstiodInfo requires namespace and pod")
 	}
 	ret := make(map[string]string)
-	for _, url := range istiodURLs {
-
-		out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, "discovery", fmt.Sprintf(`curl localhost:8080/%s`, url), p.DryRun)
+	for _, url := range common.IstiodDebugURLs(p.ClusterVersion) {
+		out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, common.DiscoveryContainerName, fmt.Sprintf(`curl localhost:8080/%s`, url), p.DryRun)
 		if err != nil {
 			return nil, err
 		}
-
 		ret[url] = out
 	}
 	return ret, nil
 }
 
+// GetNetstat returns netstat for the given container.
+func GetNetstat(p *Params) (map[string]string, error) {
+	if p.Namespace == "" || p.Pod == "" {
+		return nil, fmt.Errorf("getNetstat requires namespace and pod")
+	}
+
+	out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, common.ProxyContainerName, "netstat -natpw", p.DryRun)
```

Comment:
make sure this doesn't block anyting if it fails, to support distroless

Reply:
all of these functions run off their own goroutines so it will not block. i think this should just immediately fail, but otherwise there is a top level command timeout.